index.html

```diff
@@ -574,6 +574,30 @@ <h4>Operations</h4>
                                 of [[RFC8037]], then [= exception/throw =] a {{DataError}}.
                               </p>
                             </li>
+                            <li>
+                              <p>
+                                Let |jwkPublic| be the {{JsonWebKey/x}} field of |jwk| interpreted
```

Comment:
What do you think - would it improve reading if we move the public key part of the patch to right before "If the d field is present" ?

Reply:
@panva any opinion?

Reply:
I tend to agree with the comment in https://github.com/WICG/webcrypto-secure-curves/issues/33#issuecomment-3150406438 in that this is already covered by having a "valid" JWK for a given sub type. Given that implementations already reject exactly what is described here I find the entire addition superfluous.

Reply:
you're talking about this, right? 
>The parameter "x" MUST be present and contain the public key encoded using the base64url [[RFC4648](https://www.rfc-editor.org/rfc/rfc4648)] encoding.

from 
>interpreted according to Section 2 of [[RFC8037]] 

I would expect that the Section 2 contains the rules of decoding..